app/controllers/idv/review_controller.rb

```diff
@@ -0,0 +1,66 @@
+module Idv
+  class ReviewController < StepController
+    include IdvSession
+
+    before_action :confirm_two_factor_authenticated
+    before_action :confirm_idv_session_started
+    before_action :confirm_idv_steps_complete
+
+    def confirm_idv_steps_complete
+      redirect_to idv_finance_path unless idv_finance_complete?
+      redirect_to idv_phone_path unless idv_phone_complete?
+    end
+
+    def new
+      idv_params.symbolize_keys!
+    end
+
+    def create
+      self.idv_applicant = applicant_from_params
+      resolution = start_idv_session
+      if resolution.success
+        init_questions_and_profile(resolution)
+        redirect_on_success
+      else
+        flash[:error] = I18n.t('idv.titles.fail')
+        redirect_to idv_session_url
```

Comment:
These two can be combined in a one-liner:
```ruby
redirect_to idv_session_url, alert: t('idv.titles.fail')
```
Note that by default, `redirect_to` only accepts certain types of flash keys, namely `alert` and `notice`, but Rails allows you to define additional flash types, with the `add_flash_types` method, which can be added to ApplicationController:
```ruby
add_flash_types :error
```